.dart_tool/package_config.json

```diff
@@ -0,0 +1,176 @@
+{
```

Comment:
I dont think this whole `.dart_tool` should be in repositroy

Reply:
deleted it.